astropy/table/table.py

```diff
@@ -3857,6 +3857,13 @@ class QTable(Table):
     except that columns with an associated ``unit`` attribute are converted to
     `~astropy.units.Quantity` objects.
 
+    .. note:: When a column of ``int`` ``dtype`` is converted to ``Quantity``, its
+              ``dtype`` is converted to ``float``.
```

Comment:
We can do live links with `` `~float` ``.
I'm not a Table expert, but I think ``dtype`` is the NumPy dtype class, so that too can be linked like `` `~numpy.dtype` ``.